Esri/arcgis-maps-sdk-swift-samples
# [Fix] Set license key bug

## Description

This PR fixes a bug where the license key wouldn't be set if the `.secrets` file didn't also include an extension.

## Linked Issue(s)

- `swift/issues/6308`

## How To Test

- Remove the `ARCGIS_EXTENSION_LICENSE_KEY` from your secrets file and ensure that "Licensed for Developer Use Only" watermark still doesn't show up.
